code/__HELPERS/jobs.dm

```diff
@@ -74,88 +74,88 @@
 	)
 	return id_style[jobname] || "noname" // default: a card with no shape
 
+GLOBAL_LIST_INIT(id_to_hud, list(
```

Comment:
please put a comment that don't call this global directly unless you need the list itself as It should be called from a dedicated proc `get_hud_by_jobname()`
I know why you had to do this (get the list to tgui static), but otherwise this should be called from a proc.